app/src/main/java/com/github/libretube/constants/PreferenceKeys.kt

```diff
@@ -112,7 +112,7 @@ object PreferenceKeys {
     const val SELECTED_FEED_FILTERS = "filter_feed"
     const val FEED_SORT_ORDER = "sort_oder_feed"
     const val LOCAL_FEED_EXTRACTION = "local_feed_extraction"
-    const val LAST_FEED_REFRESH_TIMESTAMP_MILLIS = "last_feed_refresh_timestamp_millis"
+    const val LAST_LOCAL_FEED_REFRESH_TIMESTAMP_MILLIS = "last_feed_refresh_timestamp_millis"
```

Comment:
I would prefer to have this change in a separate commit, as it has nothing to do with the notification changes